microgrid_optimizer.py

```diff
@@ -963,6 +972,10 @@ def parse_results(self):
             results_summary = {key: {'mean': np.mean(val), 'std': np.std(val)}
```

Comment:
If we're calculating the max for pv, batt and gen, it probably makes sense to add the calculation here rather than have the special 'most-conservative' code below.